freqtrade/freqai/prediction_models/CatboostClassifier.py

```diff
@@ -39,3 +41,34 @@ def fit(self, data_dictionary: Dict) -> Any:
         cbr.fit(train_data)
 
         return cbr
+
+    def predict(
```

Comment:
This ain't needed for LightGBMClassifier? is this catboost specific?

Reply:
I just made a BaseClassifierModel which allows all the classifiers to get the predict_proba fun.